src/Aside/Aside.spec.tsx

```diff
@@ -0,0 +1,22 @@
+import { screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { renderWithProviders } from '../utils/testUtils';
+import Aside from './Aside';
+
+describe('Aside Component', () => {
+  describe('when initially rendered', () => {
+    it('should display the Create Event button', () => {
+      renderWithProviders(<Aside />);
+      expect(screen.getByText(/Create Event/)).toBeInTheDocument();
```

Comment:
try querying elements by their role when possible.
```suggestion
      expect(screen.getByRole('button', { name: 'Create Event' })).toBeInTheDocument();
```